src/utils/stockDataUtils/getFormattedStockExchange.test.js

```diff
@@ -0,0 +1,30 @@
+import {
+  getFormattedStockExchange
+} from './getFormattedStockExchange';
+import * as selectors from '../../client/selectors';
+
+describe('getFormattedStockExchange', () => {
+  beforeAll(() => {
+    selectors.exchangeSelector = jest.fn();
+  });
+
+  afterAll(() => {
+    jest.resetAllMocks();
+  });
+
+  it('returns TSX if state.exchange is Toronto', () => {
+    selectors.exchangeSelector.mockReturnValue('Toronto');
+    expect(getFormattedStockExchange({ exchange: 'Toronto' }))
+      .toEqual('TSX');
+  });
+  it('returns NASDAQ if state.exchange is NasdaqGS', () => {
+    selectors.exchangeSelector.mockReturnValue('NasdaqGS');
+    expect(getFormattedStockExchange({ exchange: 'NasdaqGS' }))
+      .toEqual('NASDAQ');
+  });
+  it('returns the exchange if state.exchange is anything else', () => {
+    selectors.exchangeSelector.mockReturnValue('NYSE');
```

Comment:
Shouldn't this be `.mockReturnValue('Anything else');` and expect it `toEqual('Anything else')` if you're going to pass `{ exchange: 'Anything else' }` to getFormattedStockExchange?